jupytercad_freecad/freecad/loader.py

```diff
@@ -244,6 +249,9 @@ def _fc_to_jcad_obj(self, obj) -> Dict:
             name=obj.Name,
         )
         for prop in obj.PropertiesList:
+            # Do not load Shape entry if we support that type in JupyterCAD
+            if prop == "Shape" and obj.TypeId in SUPPORTED_TYPES:
+                continue
             prop_type = obj.getTypeIdOfProperty(prop)
```

Comment:
We can be even more lazy and only load properties we understand here. E.g. for the box we only understand the properties:
```python
from jupytercad_core.schema.interfaces.box import IBox
list(IBox.model_fields.keys())
```

Reply:
The issue is the exported JCAD file contains an unsupported "shape" value in the `obj_data` dict, not the "Shape" key in the `parameters` key

https://github.com/jupytercad/JupyterCAD-FreeCAD/blob/273a0751116e64b03ce7dbce3153dcfd7b635188/jupytercad_freecad/freecad/loader.py#L240-L245

Reply:
I am not sure what you are commenting here? Are you commenting my comment or the code change?